tokens/figma/utils/hex-to-rgba.mjs

```diff
@@ -0,0 +1,28 @@
+export function hexToRgba(hex, alpha = 1) {
+  // Remove the leading # if it's present
+  hex = hex.replace(/^#/, "");
+
+  // Expand shorthand form (e.g., "03F") to full form (e.g., "0033FF")
+  if (hex.length === 3) {
+    hex = hex
+      .split("")
+      .map((char) => char + char)
+      .join("");
+  }
+
+  // Parse the r, g, b values
+  const bigint = parseInt(hex, 16);
```

Comment:
```suggestion
  const bigInt= parseInt(hex, 16);
```
maybe?

Reply:
Prettier just give here a space, i would mind meanwhile the function works

Reply:
oh! I was referring about the camelCase format. I delete the space between "t" and "=" by mistake